dts/arm/realtek/ec/rts5912-pinctrl.dtsi

```diff
@@ -0,0 +1,645 @@
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ i2c06_data_gpio037: i2c06_data_gpio037 {
+			pinmux = <REALTEK_RTS5912_PINMUX(37, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+
+	/omit-if-no-ref/ i2c07_clk_gpio038: i2c07_clk_gpio038 {
+			pinmux = <REALTEK_RTS5912_PINMUX(38, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ i2c07_data_gpio039: i2c07_data_gpio039 {
+			pinmux = <REALTEK_RTS5912_PINMUX(39, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/* I2C PINCTRL SETTING END */
+	/* JTAG PINCTRL SETTING START */
+	/omit-if-no-ref/ jtag_tdi_gpio87: jtag_tdi_gpio87 {
+			pinmux = <REALTEK_RTS5912_PINMUX(87, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ jtag_tdo_gpio88: jtag_tdo_gpio88 {
+			pinmux = <REALTEK_RTS5912_PINMUX(88, REALTEK_RTS5912_FUNC3)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ jtag_rst_gpio89: jtag_rst_gpio89 {
+			pinmux = <REALTEK_RTS5912_PINMUX(89, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ jtag_clk_gpio90: jtag_clk_gpio90 {
+			pinmux = <REALTEK_RTS5912_PINMUX(90, REALTEK_RTS5912_FUNC2)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ jtag_tms_gpio91: jtag_tms_gpio91 {
+			pinmux = <REALTEK_RTS5912_PINMUX(91, REALTEK_RTS5912_FUNC2)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/* JTAG PINCTRL SETTING END */
+	/* KSM PINCTRL SETTING START */
+	/* KSO PINCTRL SETTING START */
+	/omit-if-no-ref/ kso0_gpio041: kso0_gpio041 {
+			pinmux = <REALTEK_RTS5912_PINMUX(41, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ kso1_gpio042: kso1_gpio042 {
+			pinmux = <REALTEK_RTS5912_PINMUX(42, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ kso2_gpio043: kso2_gpio043 {
+			pinmux = <REALTEK_RTS5912_PINMUX(43, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ kso3_gpio044: kso3_gpio044 {
+			pinmux = <REALTEK_RTS5912_PINMUX(44, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+
+	/omit-if-no-ref/ kso4_gpio045: kso4_gpio045 {
+			pinmux = <REALTEK_RTS5912_PINMUX(45, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ kso5_gpio046: kso5_gpio046 {
+			pinmux = <REALTEK_RTS5912_PINMUX(46, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ kso6_gpio047: kso6_gpio047 {
+			pinmux = <REALTEK_RTS5912_PINMUX(47, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+	/omit-if-no-ref/ kso7_gpio048: kso7_gpio048 {
+			pinmux = <REALTEK_RTS5912_PINMUX(48, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
+	};
+
+	/omit-if-no-ref/ kso8_gpio049: kso8_gpio049 {
+			pinmux = <REALTEK_RTS5912_PINMUX(49, REALTEK_RTS5912_FUNC1)>;
```

Comment:
suggestion, can be simplified to `<REALTEK_RTS5912_PINMUX(49, FUNC1)>;` using token pasting

Reply:
We have replaced `REALTEK_RTS5912_FUNC1` with `FUNC1`. We kept `REALTEK_RTS5912_FUNC1` in `realtek-rts5912-pinctrl.h` and used macro to define `FUNC1`.